requirements_doc.txt

```diff
@@ -8,7 +8,7 @@ memory_profiler
 neo
 seaborn!=0.11.2
 sphinx_copybutton
-mne-bids
+https://github.com/mne-tools/mne-bids/zipball/2bb880a231ec52e85f4033e3782b0d9af4d510ad
```

Comment:
this can maybe be fixed now to using main branch

Reply:
Or even the latest release?